student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(id, classroom, name: 'Unknown', age: nil, parent_permission: true)
-    super(id, name: name, age: age, parent_permission: parent_permission)
+  def initialize(id, classroom, name = 'Unknown', age: nil, parent_permission: true)
+    super(id, name, age: age, parent_permission: parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ] Kindly add the method for the association with the classroom as hinted by the first reviewer.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```